### PR TITLE
Sync GCP.AuditLog with enterprise

### PR DESCRIPTION
**Pull request by alxarch**
## Background

GCP.AuditLog was out of sync with enterprise

### CODE REVIEW

File: internal/log_analysis/log_processor/parsers/gcplogs/gcplogs.go [hunk 1]
```diff
@@ -43,6 +43,8 @@ var logTypes = logtypes.Must(LogTypePrefix, logtypes.Config{
 	Name: TypeAuditLog,
 	Description: `Cloud Audit Logs maintains three audit logs for each Google Cloud project, folder, and organization: Admin Activity, Data Access, and System Event.
 Google Cloud services write audit log entries to these logs to help you answer the questions of "who did what, where, and when?" within your Google Cloud resources.
+
+Panther Enterprise Only
```
Comment (kostaspap) [inline internal/log_analysis/log_processor/parsers/gcplogs/gcplogs.go:47]: Actually this is no longer true.

Comment (alxarch) [reply internal/log_analysis/log_processor/parsers/gcplogs/gcplogs.go:47]: Don't we need a puller to fetch this?

Comment (kostaspap) [reply internal/log_analysis/log_processor/parsers/gcplogs/gcplogs.go:47]: Nope

Comment (alxarch) [reply internal/log_analysis/log_processor/parsers/gcplogs/gcplogs.go]: removed

**Review by kostaspap — APPROVED** [review]

**Comment by alxarch** [comment]
some weird failure in CI
https://app.circleci.com/pipelines/github/panther-labs/panther/9631/workflows/f61d04c1-7232-4162-87bb-e20ef81f9b55/jobs/31461/parallel-runs/0/steps/0-104?invite=true

**Pull request merged**